src/status_im/ui/components/tooltip/styles.cljs

```diff
@@ -23,7 +23,7 @@
    :align-items :center
    :left        0
    :right       0
-   :bottom      bottom-value
+   :transform   [{:translateY bottom-value}]
```

Comment:
For perf?

Reply:
Yes, for perf. `:bottom` can't be animated using native driver, thus animation works in js thread, loads at with unnecessary calculations and also looks laggy at some circumstances.

Reply:
I changed it a bit so that instead of adding new magic number we only change number's sign :D